code/modules/cm_marines/orbital_cannon.dm

```diff
@@ -493,11 +544,13 @@ var/list/ob_type_fuel_requirements
 		for(var/k = 1 to instant_amount)
 			var/turf/U = pick(turf_list)
 			fire_in_a_hole(U)
+
 		sleep(delay_between_clusters)
 
 /obj/structure/ob_ammo/warhead/cluster/proc/fire_in_a_hole(turf/loc)
 	new /obj/effect/overlay/temp/blinking_laser (loc)
 	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(cell_explosion), loc, explosion_power, explosion_falloff, EXPLOSION_FALLOFF_SHAPE_LINEAR, null, create_cause_data(initial(name), source_mob)), 1 SECONDS)
+	addtimer(CALLBACK(src, PROC_REF(handle_ob_shake), loc), 1 SECONDS)
```

Comment:
This is throwing a bunch of runtimes about callback to a qdeleted object. Gonna need a fix before merge.

Reply:
Should be resolved now.